### PR TITLE
Bumps in garden : ign-launch6

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/685

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 Build | Status
 -- | --
-Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-launch/branch/ign-launch5/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-launch)
-Ubuntu Focal | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_launch-ci-ign-launch5-focal-amd64)](https://build.osrfoundation.org/job/ignition_launch-ci-ign-launch5-focal-amd64)
-Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_launch-ci-ign-launch5-homebrew-amd64)](https://build.osrfoundation.org/job/ignition_launch-ci-ign-launch5-homebrew-amd64)
+Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-launch/branch/ign-launch6/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-launch)
```
Comment (chapulina) [inline README.md:12]: Ah we should have changed this to `main` a while back

```suggestion
Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-launch/branch/main/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-launch)
```

File: README.md [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
+Ubuntu Focal | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_launch-ci-ign-launch6-focal-amd64)](https://build.osrfoundation.org/job/ignition_launch-ci-ign-launch6-focal-amd64)
```
Comment (chapulina) [inline README.md:13]:
```suggestion
Ubuntu Focal | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_launch-ci-main-focal-amd64)](https://build.osrfoundation.org/job/ignition_launch-ci-main-focal-amd64)
```

File: README.md [hunk 3]
```diff
@@ -9,9 +9,9 @@
...
+Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_launch-ci-ign-launch6-homebrew-amd64)](https://build.osrfoundation.org/job/ignition_launch-ci-ign-launch6-homebrew-amd64)
```
Comment (chapulina) [inline README.md:14]:
```suggestion
Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_launch-ci-main-homebrew-amd64)](https://build.osrfoundation.org/job/ignition_launch-ci-main-homebrew-amd64)
```

**Review by chapulina — APPROVED** [review]
Just some minor comments that aren't really this PR's fault, but pre-existing issues that were surfaced by it.

LGTM otherwise, once it has 🟢 CI.

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Comment by chapulina** [comment]
Windows has some failed tests which I think are expected.

Homebrew is broken because Gazebo is broken.

Merging!

**Pull request merged**